### ISSUE TITLE
Any update on the situation?

### ISSUE THREAD

**Issue by TheVolecitor** [Lomusire/unlimited-cloud-storage#18]

**Comment by Ctmax-ui** [comment]
Yes, terabox removed reward system and etc, so our extension is useless now

**Comment by BabakarSyr** [comment]
They did it because of your extension or it has nothing to do with it.

**Comment by Ctmax-ui** [comment]
> They did it because of your extension or it has nothing to do with it.

na bro we sure, its because of our extension, recently I found our extension link on form website in dk web.

**Comment by BabakarSyr** [comment]
I see, it's too bad!! I think that next time it's better to avoid publishing it on the chrome store because it made your extension known.

**Comment by Ctmax-ui** [comment]
maybe who knows

**Comment by Cristofer543** [comment]
> maybe who knows

True for the next one don't take her up to the store and keep her from becoming so famous

**Comment by Ctmax-ui** [comment]
> True for the next one don't take her up to the store and keep her from becoming so famous

can do that, but who knows future it still can be accessed by GitHub, so the extension still in public.
